README.org

```diff
@@ -49,15 +49,68 @@ or if you use ~straight~ then
   #+BEGIN_SRC emacs-lisp
     (setq expenses-currency "$")
   #+END_SRC
+
+* Importing Bank Statements
+
+You can import directly from Bank CSV exports.
+
+Example CSV export from a bank
+: Bank Account,Date,Narrative,Debit Amount,Credit Amount,Balance,Categories,Serial
+: 5544,05/12/2021,"THAT CAFE SYDNEY AUS",4.00,,0.00,OTHER,
+
+- *Simple import use*
+  ~expenses-import-expense-with-bank-profile~
+
+  First look at the CSV to determine the order remember that it starts at zero for the first column.
+
 - *Bank profiles*
+
   To be able to import expenses from bank statements conveniently, you can set profiles
   for your banks using the varibale ~expenses-bank-profiles~. This is a list where each element
   is of the form ~(bank-name sep date-col debit-col date-format narrative-col category-col)~
   #+BEGIN_SRC emacs-lisp
-    (setq expenses-bank-profiles (("TRUSTED-BANK" "," 1 2 "dd/mm/yyyy" 3 4)
+    (setq expenses-bank-profiles (("EXAMPLE-BANK" "," 1 3 "dd/mm/yyyy" 2 -1)
 				  ("ANOTHER-BANK" "\t" 1 4 "yyyy/mm/dd" 2 3)))
   #+END_SRC
-  
+  NOTE: If category is set to -1 then you will be prompted to assign categories for each line.
+
+- *Auto assign categories*
+
+  You can auto assign based on keyword or phrase. To set this up you
+  can override the default phrases and keywords in your init.el file
+  with the following;
+
+  #+begin_src emacs-lisp
+  (require 'ht)
+
+  (setq expenses-utils-phrases-alist '(
+                                       ("BUDGET DIRECT" . "Car")
+                                       ("WW METRO" . "Petrol")
+                                       ("RS COMPONENTS" . "Electronics")
+                                       ))
+
+  (defcustom expenses-utils-keyword-category-ht (ht
```

Comment:
`defcustom` is used to define a customizable variable. To set a value just use `(setq variable value)`. For example, Here it should be like below
```
(setq expenses-utils-keyword-category-ht (ht ("keyword" "Category")))
```